Sources/xcodeproj/Scheme/XCScheme+LaunchAction.swift

```diff
@@ -272,40 +272,5 @@ extension XCScheme {
             }
             return element
         }
-
-        // MARK: - Equatable
-
-        override func isEqual(to: Any?) -> Bool {
```

Comment:
Am I wrong in assuming that the conformance to `Equatable` is not generated automatically by the compiler?

Reply:
Not for classes and tuples: https://github.com/apple/swift-evolution/blob/master/proposals/0185-synthesize-equatable-hashable.md#synthesis-for-class-types-and-tuples.

A number of these classes could probably be structs though.